frontend/www/js/omegaup/components/homepage/Slide.vue

```diff
@@ -52,6 +52,10 @@ export default class Slide extends Vue {
     height: 500px;
   }
 
+  @media only screen and (max-width: 767px) {
+    height: 700px;
+  }
```

Comment:
```suggestion
  height: 700px;
```

No necesitas crear otra media query. Y de preferencia pon esta regla antes de la media query de la L51.